2016/proposals/stolliker-ryan.md

```diff
@@ -0,0 +1,67 @@
+# Ryan Stolliker: Result Aggregation Server for Software Carpentry
+
+## Abstract
+
+During Software Carpentry's workshops which teach computing skills to the greater scientific community, learners use Python scripts to test that they have installed the programming environment and other dependencies correctly. Currently, this information is known only to the participants and cannot be used or tracked by the organizers. Thus, the goal of this project is to enable the collection of this installation data and to store it for future use and analysis.
+
+## Technical Details
```

Comment:
Yes, I think VM is a fine solution (it may change some output from `platform` should you use it [https://github.com/numfocus/gsoc/pull/111/files#r56413805]) + I think me and @rgaiacs are running Linuxes so we can help with testing.